java/ql/lib/semmle/code/java/frameworks/kotlin/StdLibGenerated.qll

```diff
@@ -1718,11 +1777,13 @@ private class StdLibGeneratedSummaryCsv extends SummaryModelCsv {
         "kotlin.time;Duration;false;div;(int);;Argument[-1];ReturnValue;taint;generated",
         "kotlin.time;Duration;false;getAbsoluteValue;();;Argument[-1];ReturnValue;taint;generated",
         "kotlin.time;Duration;false;minus;(Duration);;Argument[-1];ReturnValue;taint;generated",
+        "kotlin.time;Duration;false;plus;(Duration);;Argument[-1];ReturnValue;taint;generated",
         "kotlin.time;Duration;false;plus;(Duration);;Argument[-1];ReturnValue;value;generated",
         "kotlin.time;Duration;false;times;(double);;Argument[-1];ReturnValue;taint;generated",
         "kotlin.time;Duration;false;times;(int);;Argument[-1];ReturnValue;taint;generated",
         "kotlin.time;DurationKt;false;times;(double,Duration);;Argument[1];ReturnValue;taint;generated",
         "kotlin.time;DurationKt;false;times;(int,Duration);;Argument[1];ReturnValue;taint;generated",
+        "kotlin.time;TimeMark;true;minus;(Duration);;Argument[-1];ReturnValue;taint;generated",
```

Comment:
Based on the below two lines, one would expect to also see:
```
"kotlin.time;TimeMark;true;minus;(Duration);;Argument[0];ReturnValue;taint;generated",
```
This line is not generated, because we don't track flow through unary minus of double.